dbreps2/src/main.rs

```diff
@@ -22,6 +32,13 @@ macro_rules! run {
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    let args = Args::parse();
+    if args.report == "" {
```

Comment:
And now that args.report is Option<String>, you can do:

```
    match args.report {
        Some(report) => println!("Hello {}!", report),
        None => println!("Hello, report arg not set..."),
    }
```

Reply:
Nice, thanks.